data/hai missions.txt

```diff
@@ -1061,3 +1061,252 @@ mission "Hai Honeymoon"
+				accept
+	npc accompany save
+		personality escort timid
+		government "Hai"
+		ship "Heavy Shuttle" "Acorn Chaser"
+	on visit
+		dialog `You have reached <planet>, but you left Nanachi's ship behind! Better depart and wait for her to arrive in this star system.`
+	on complete
+		conversation
+			`You help the family gather all their luggage and bid them farewell, but right as they disappear into a hallway Nanachi lets out a heavy sigh. "I bit off more than I could chew. Those kids kept wanting to play with the ship's controls, and one of them tried to go up the air vent in a game of hide-and-seek. I couldn't get any rest since I had to keep making sure they weren't about to dump their own bags into the void by accident."`
+			`	"Do you want to get some sleep before trying to find another mission?" you ask, hoping she won't try to overwork herself. "Yes, I could really use that. Sorry to make you wait again captain <last>. I'll rest a bit, see if I can't find a tourist spot nearby to relax, and meet up with you the in spaceport after a while."`
+
+
+
+mission "Nanachi 3"
+	name "Deliver spices with Nanachi"
+	description "Accompany Nanachi to Alta Hai to pick up some Quarg spices for a festival in <planet>."
+	source "Hai-home"
+	stopover "Alta Hai"
+	destination "Allhome"
+	deadline 11
+	to offer
+		has "Nanachi 2: done"
+	on offer
+		conversation
+			`You send a message to Nanachi and you two meet up in the spaceport. You're still walking toward the job board, when she asks: "Back home I heard stories about how humans have a much larger population than the Hai, and that their home planet has even more people than Hai-home, is it true that there are many other human worlds with spaceports this large?" You name her a few human planets that are extremely populous, and how even some planets with fewer people than this world have incredibly large spaceports.`
+			`	While you tell her stories of many different planets you've visited, she seems to be particularly interested in Earth. "I've seen a bit of human architecture in videos of planets like Allhome, but I can't even imagine what your capital must look like. Tens of thousands of years of another species' history, it sounds even more exciting than what I heard about our landmarks in Hai-home. Captain, do you think that someday, when we no longer need to be kept hidden from humanity, that I'll be able to visit Earth, and see the beauty of your capital?" Seeing how little she knows about humanity, it's not surprising that she is unaware of the state that Earth is in.`
+			choice
+				`	(Avoid talking about Earth.)`
+					goto avoid
+				`	(Lie about Earth.)`
+					goto fake
+				`	(Tell her about how Earth actually is.)`
```

Comment:
I feel like these would be more impactful as dialog. Perhaps something along the lines of the following:
```
`	"I've never actually been to Earth before, so I couldn't tell you what it's like"`
`	"Earth is a beautiful place, so it would be nice if you could see it."`
`	"Earth isn't actually that great a place to visit. Most of its inhabitants would take any change they're given to leave it."`
```
The beginning of `label avoid` would need to be changed, but `label fake` and `label truth` look like they wouldn't need to be change with because of the dialog of the choice.

Reply:
I feel like, if this changed, then the conversations being led into this need to be changed, and it ends up more as a detailed history lesson/lie than anything.

As of right now, I'm going to resolve this as-is/no change.  If people playing give feedback that they want it changed, I'm more than happy to look at it again.